_content/references.md

```diff
@@ -1,5 +1,155 @@
 ## References
 
+### Word list
+
+This section lists commonly used words and phrases in Agora technical documentation. For the usage of other words, refer to the [word list](https://developers.google.com/style/word-list) in Google developer documentation style guide.
+
+#### Agora-related naming conventions
+
+When referring to an Agora SDK, use "the". For example, "the Agora Native SDK" or "the Agora Recording SDK", and not "Agora Signaling SDK". 
+
+Do not use "the" for proper nouns such as "Agora Video Call" or "Agora Recording". Do not use "the Agora Cloud Recording".
+
+| **Full Name**                                | **Short Form**             | **Description/Notes**                                        |
+| -------------------------------------------- | -------------------------- | ------------------------------------------------------------ |
+| Agora Analytics                              | Agora Analytics            | A tool that tracks and analyzes the usage and quality of calls. |
```

Comment:
Can you add all the products to this list please.

Reply:
Done.